<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GreenmileConfigWidget</class>
 <widget class="QWidget" name="GreenmileConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="17" column="0">
      <widget class="QLabel" name="censusConnectionFreqLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Census Connection Frequency (ms)</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QSpinBox" name="censusConnectionFreqMSSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9000</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QSpinBox" name="censusRequestTimeoutSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="censusMaxActiveConnectionsLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Census Maximum Number of Active Connections</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="connectionFreqLabel">
       <property name="text">
        <string>Greenmile Connection Frequency (ms)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="censusRequestTimeoutLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Census Request Timeout in Seconds</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLineEdit" name="censusAddressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>https://geocoding.geo.census.gov</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="censusAddressLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Census Server Address</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="useCensusCheckBox">
       <property name="text">
        <string>Use US Census Geocoder</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QPushButton" name="saveSettingsButton">
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="connectionFreqMSSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="text">
        <string>password</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="serverAddressLabel">
       <property name="text">
        <string>Greenmile Server Address</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="requestTimeoutSpinbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9001</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="serverAddressLineEdit">
       <property name="text">
        <string>https://charliesproduce.greenmile.com</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="requestTimeoutLabel">
       <property name="text">
        <string>Greenmile Request Timeout in Seconds</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Greenmile User Name</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Greenmile Password</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="maxActiveConnectionsLabel">
       <property name="text">
        <string>Greenmile Maximum Number of Active Connections</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QSpinBox" name="maxActiveConnectionsSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QSpinBox" name="censusMaxActiveConnectionsSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9001</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serverAddressLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>requestTimeoutSpinbox</tabstop>
  <tabstop>connectionFreqMSSpinBox</tabstop>
  <tabstop>maxActiveConnectionsSpinBox</tabstop>
  <tabstop>saveSettingsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
